subjects/graphql/README.md

```diff
@@ -39,18 +41,29 @@ Here are some possible combinations for the creation of the graphs:
 
 Any other information you desire to display is welcome and will be noted.
 
-### Hosting
+### Login Page
 
-Besides the creation of your own profile you will have to host it! There are several places where you can host your profile,\
-for example: [github-pages](https://pages.github.com/), [netlify](https://www.netlify.com/) and so on. You are free to choose the hosting place.
+You'll need a [JWT](https://jwt.io/introduction) to access the GraphQL API. A JWT can be obtained from the `signin` endpoint (`https://((DOMAIN))/api/auth/signin`).
+
+You may make a `POST` request to the `signin` endpoint, and supply your credentials using `Basic` authentication, with base64 encoding.
+
+Your login page must function with both:
+
+- username:password
+- email:password
 
----
+If the credentials are invalid, an appropriate error message must be displayed.
 
-### Usage
+You must provide a method to log out.
 
-> To test your queries you can access the GraphQL IDE on _https://((DOMAIN))/graphiql/_ or create your own **GraphiQL Docs**. This will give you a bigger picture of the tables, attributes and all the types of queries that you can do.
+When making GraphQL queries, you'll supply the JWT using `Bearer` authentication.
 
-Here are the list of tables that you are allowed to query (it will be only provided the columns present on the tables):
+### Hosting
+
+Besides the creation of your own profile you will have to host it. There are several places where you can host your profile,\
+for example: [github-pages](https://pages.github.com/), [netlify](https://www.netlify.com/) and so on. You are free to choose the hosting place.
+
```

Comment:
Currently we do not set the right headers in the `sign-in` route to allow for external use.
![image](https://user-images.githubusercontent.com/231748/228796101-e562d9cb-ed39-473b-ac3d-1ce0c9cf0436.png)

We should allow it, I'll create a linked PR on `01-edu/all`